applications/modal-bi-encoder/app.py

```diff
@@ -0,0 +1,23 @@
+from fastapi import FastAPI
+from src.main import (
+    encode,
+    EncodeInput,
+    EncodeOutput,
+    cosine_similarity,
+    CosineSimilarityInput,
+    CosineSimilarityOutput,
+)
+
+app = FastAPI(
+    title="Sentence Transformer: Bi-Encoder",
+)
+
+
+@app.post("/encode", response_model=EncodeOutput)
```

Comment:
this should look like the openai embedding spec https://platform.openai.com/docs/api-reference/chat/create

Reply:
Changed the `encode` endpoint to be more like the openai embeddings endpoint https://platform.openai.com/docs/api-reference/embeddings/create
* renamed to `embeddings`
* changed types to model openai output, inspired from this file: <https://github.com/jxnl/fastllm/blob/main/applications/embeddings-gpt4all/src.py>

OpenAPI docs examples with new endpoints: 
<img width="1192" alt="image" src="https://github.com/jxnl/fastllm/assets/54543035/e8cda421-6b26-462e-a589-8c64451c5e02">
<img width="1204" alt="image" src="https://github.com/jxnl/fastllm/assets/54543035/10a16a1c-cd61-408e-bf25-a7b74412b005">

Reply:
I also updated the README to reflect these changes